examples/karaf-jersey-example/karaf-jersey-example.servicedef/src/main/java/org/apache/karaf/examples/jersey/servicedef/beans/Count.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements.  See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.
+ *  The ASF licenses this file to You under the Apache License, Version 2.0
+ *  (the "License"); you may not use this file except in compliance with
+ *  the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.apache.karaf.examples.jersey.servicedef.beans;
```

Comment:
No need to put in `beans`. The POJO used by the service can be at the same service definition level aka `org.apache.karaf.examples.jersey.service`.

Reply:
I've used org.apache.karaf.examples.jersey.services for the package containing the service implementation, and org.apache.karaf.examples.jersey.servicedef for the service implementation (which also has a match in part of the artifactId, the module name and bundle-symbolic-name).

Is it OK to keep this? 

Or will the service definition package need to be changed to org.apache.karaf.examples.jersey.service and the implementation package need to be changed to something else?